### ISSUE TITLE
Disconnected from server: Poor connection.

### ISSUE THREAD

**Issue by mblogest** [RiptideNetworking/Riptide#159]
Somtimes Disconnected from server: Poor connection. But Event Diddisconnect() Not Work and callback

**Comment by tom-weiland** [comment]
You mean the client's `Disconnected` event just isn't invoked? Are you calling the server's and client's `Update` methods regularly?

**Comment by SheebSuka** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by mblogest** [comment]
yes i use Server.Update(); in FixedUpdate(). but sometimes diddisconnect not call. when i write any debug.log() in diddisconnect not work. but this log(Disconnected from server: Poor connection) show sometime and disconnect client from sever

**Comment by tom-weiland** [comment]
> yes i use Server.Update(); in FixedUpdate().

What about `Client.Update()`? You need both for it to work.

> when i write any debug.log() in diddisconnect not work. but this log(Disconnected from server: Poor connection) show sometime and disconnect client from sever

Does your custom log in the `Disconnected` event *ever* get called? Is there any sort of pattern regarding when the event does work and whether or not "Disconnected from server: Poor connection" is logged?

**Comment by mblogest** [comment]
Yes i use update in both side. but no, nothing is shown on the client side. no error is shown.

**Comment by tom-weiland** [comment]
@mblogest can you join the Discord server and make a post in the help channel there, including any relevant code, logs, and other details about your setup? Troubleshooting something like this will be easier on Discord than here on GitHub.

**Comment by tom-weiland** [comment]
Closing as stale.
